charts/cert-manager-1.14.5/values.yaml

```diff
@@ -126,7 +126,7 @@ maxConcurrentChallenges: 60
 image:
   # The container registry to pull the manager image from
   # +docs:property
-  # registry: quay.io
+  # registry: container-registry.oracle.com/olcne
```

Comment:
do we want /olcne on all these registry entries, since the images all have olcne/ prefix.  There are a bunch like this.